crates/bevy_render/src/camera/camera.rs

```diff
@@ -316,6 +360,11 @@ impl Camera {
     /// and between 0.0 and 1.0 on the Z axis.
     /// To get the world space coordinates with the viewport position, you should use
     /// [`world_to_viewport`](Self::world_to_viewport).
+    ///
+    /// Returns `None` if any of these conditions occur:
+    /// - The projection matrix is invalid or cannot be inverted
+    /// - The camera transform is invalid
+    /// - The ndc is invalid
```

Comment:
Same as https://github.com/bevyengine/bevy/pull/8841/files#r1246420948 except here if the Matrix is not set it would panic or put invalid values (depending on glam_assert) too because the 0 matrix can't be inverted.

The "quick fix" to return None early could be added for this function too.

Reply:
Have also revised this